endoreg_db/views/anonymization/validate.py

```
@@ -0,0 +1,107 @@
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from endoreg_db.models import RawPdfFile, VideoFile
+from endoreg_db.serializers.anonymization import SensitiveMetaValidateSerializer
+
+
+logger = logging.getLogger(__name__)
+
+
+class AnonymizationValidateView(APIView):
```

Pylint (4.0.1)

[refactor] Too few public methods (1/2)

(R0903)